### ISSUE TITLE
Preserve attributes

### ISSUE THREAD

**Issue by danmarshall** [elrumordelaluz/path-that-svg#1]
Hello, thanks for this library, I was just about to write something similar 👍 
I'm having an issue where I would like my `class` attribute to be preserved:
```
<svg><line class="foo" x1="0" y1="0" x2="5" y2="5" /></svg>
```
becomes:
```
<svg><path d="M0 0 L5 5" /></svg>
```
The solution does not need to be limited to just the `class` attribute, I suppose any attribute that is not used by the conversion and does not collide with `d` should carry over.

**Comment by elrumordelaluz** [comment]
hey @danmarshall you are absolutely right, thank you! 

I did a very fast cleanup, instead I need to [delete](https://github.com/elrumordelaluz/path-that-svg/blob/master/index.js#L13) only those attributes not used by the new `path` element but used by its element before the [conversion](https://github.com/elrumordelaluz/path-that-svg/blob/master/index.js#L9)

**Comment by elrumordelaluz** [comment]
Should work as expected in `path-that-svg@1.2.1`. 
Please let me know if there's any related _issue_.

Thanks again,

**Comment by danmarshall** [comment]
Hi Lionel, it seems there is another issue: *transform* attributes are also dropped 😯
```
<svg><path class="c1" d="M0,0 V1" transform="translate(9 9)" /></svg>
```
becomes
```
<svg><path class="c1" d="M0,0 V1"/></svg>
```

**Comment by elrumordelaluz** [comment]
hey @danmarshall thank you again to report this. 

Should be solved in `v1.2.2`.
